ui.go

```diff
@@ -875,8 +892,9 @@ func (ui *ui) drawStatLine(nav *nav) {
 			}
 		case "ind":
 			ruler = append(ruler, fmt.Sprintf("%d/%d", ind, tot))
+		default:
+			ruler = addStatLineOpt(ruler, s, opts[s])
```

Comment:
Minor: I would do

```
if value, ok:= opts[s]; ok {
    ruler = addStatLineOpt(ruler, s, value)
}
```

and get rid of the `if val=""` check in `addStatLineOpt`. I don't know how useful this is, but it would matter e.g. if somebody wants to show the value of the `ifs` option for some reason. Also, it makes the code a bit easier to read.

Reply:
Good point about the `ifs` option, I have fixed this in 21b799c. Thanks for the suggestion.

Reply:
Sounds great, thanks!